pkg/postage/batchstore/mock/store.go

```diff
@@ -13,21 +13,23 @@ import (
 	"github.com/ethersphere/bee/pkg/postage"
 	"github.com/ethersphere/bee/pkg/postage/batchstore"
 	"github.com/ethersphere/bee/pkg/storage"
+	"github.com/ethersphere/bee/pkg/swarm"
 )
 
 var _ postage.Storer = (*BatchStore)(nil)
 
 // BatchStore is a mock BatchStorer
 type BatchStore struct {
-	rs                *postage.ReserveState
-	cs                *postage.ChainState
-	id                []byte
-	batch             *postage.Batch
-	getErr            error
-	getErrDelayCnt    int
-	updateErr         error
-	updateErrDelayCnt int
-	resetCallCount    int
+	rs                    *postage.ReserveState
+	cs                    *postage.ChainState
+	isWithinStorageRadius bool
```

Comment:
Instead of a specific value, I would recommend passing the radius checker interface. This will provide more flexibility.